src/server/api/controllers/events.js

```diff
@@ -77,7 +84,7 @@ export default class EventsController {
       .then((response) => response.json())
       .then((events) => {
         res.send({ list: events.data.allEvents.sort((a, b) =>
```

Comment:
I'm putting this one on hold for now - at the moment if `startDateTime` is missing, the whole site will explode. We should handle this in a clever way, probably on BB level.